### PR TITLE
Propose RFC: Test suite

### PR DESCRIPTION
**Pull request by dhh1128**

### CODE REVIEW

File: features/0270-protocol-test-suite/README.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2019-10-25
+- Status Note: Codifies some thinking about scope and mental model that are already socialized. Provides some new thinking as well.
+- Supersedes: Partially, and in some ways, [Indy HIPE 0015](https://github.com/hyperledger/indy-hipe/blob/master/text/0015-agent-test-suite-interface/README.md) and [Indy HIPE 0016](https://github.com/hyperledger/indy-hipe/blob/master/text/0016-agent-test-suite-v1/README.md). Also, represents answers to questions that the community first posed in [this HackMD doc](https://hackmd.io/JW5b9xYCRGKqyqhVevTZ_g) and first attempted to answer in the [indy-agent repo](https://github.com/hyperledger/indy-agent).
+- Start Date: 2018-10-25
+- Tags: feature
+
+## Summary
+
+Describes the goals, scope, and interoperability contract of the Aries Protocol Test Suite. Does NOT serve as a design doc for the test suite code, or as a developer guide explaining how the test suite can be run; see the [test suite odebase](https://github.com/hyperledger/aries-protocol-test-suite) for that.
```
Comment (kdenhartog) [inline features/0270-protocol-test-suite/README.md]: odebase -> codebase

**Comment by dhh1128** [comment]
Yeah, maybe it belongs in concepts instead of features. I can move it. Don't merge until I do.

File: features/0270-protocol-test-suite/README.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+   
+2. **Describe results in a formal, granular, reproducible way** that supports comparison between agents A and B, and between A at two different points of time or in two different configurations.
+
+    This implies a structured report, as well as support for versioning of the suite, the agents under test, and the results.
+
+3. **Track the collective community state of the art**, so measurements are comprehensive and up-to-date, and so new ideas automatically encounter pressure to be vetted for interoperability.
+ 
+    The test suite isn't a compliance tool, and it should be unopinionated about what's important and what's not. However, it should embody a broad list of testable features--[certainly, ones that are standard](/README.md#accepted), and often, [ones that are still maturing](/README.md#demonstrated).
```
Comment (llorllale) [inline features/0270-protocol-test-suite/README.md]: > The test suite isn't a compliance tool

This is an odd statement to include in an RFC for a test suite with the ultimate goal of ensuring interoperability.

Comment (kdenhartog) [reply features/0270-protocol-test-suite/README.md]: My understanding of this statement is that he's saying there's no set of required protocols that must be supported in order to be "compliant". E.g. an implementer may choose to not support the trust ping protocol, which doesn't make them non-compliant.

**Comment by kdenhartog** [comment]
@dhh1128  I hit a weird bug with github, where it was asking me update branch, failed, and had a try again button. Turns out this merged it somehow, so I'll submit a PR now to move this to concepts rather than features.

**Pull request merged**